source/voice_control/create_wake_word.markdown

```diff
@@ -0,0 +1,105 @@
+---
+title: "Create your own wake word"
+---
+
+You can now create your own wake word to use with Home Assistant. The procedure below will guide you to train a model. The model is trained using voice clips generated by our local neural text-to-speech system [Piper](https://github.com/rhasspy/piper).
+
+_Want to now more about how this all works? Check out the [openWakeWord](https://github.com/dscripka/openWakeWord) project by David Scripka.)_
+
+Depending on the word, training a model on your own wake word may take a few iterations and a bit of tweaking. This guide will take you through the process step by step.
+
+## Prerequisites
+
+- latest version of Home Assistant
+- [M5Stack ATOM Echo Development Kit](https://shop.m5stack.com/products/atom-echo-smart-speaker-dev-kit?ref=NabuCasa)
+- successfully completed the [$13 voice remote for Home Assistant](/voice_control/thirteen-usd-voice-remote/) tutorial
+
+## To create your own wake word
+
+1. Think of a wake word.
+   - A word or short phrase (3-4 syllables) that is not commonly used so that it does not trigger Home Assistant by mistake.
+   - Currently, only wake words in English are supported.
+2. Open the [wake word training environment](https://colab.research.google.com/drive/1q1oe2zOyZp7UsB3jJiQ1IFn8z5YfjwEb?usp=sharing#scrollTo=1cbqBebHXjFD).
```

Comment:
Should we mention that Google is offering limited free compute, but that it should be enough to train a wake word ?

Reply:
If we expect there to be enough resources, can we not clutter the procedure with informational content? 
This info is mentioned in the troubleshooting section.
If we think there is a considerable risk that it won't run, we could add a link to the troubleshooting?